test/models/hold_change_test.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class HoldChangeTest < ActionController::TestCase
+
+  setup do
+    @hold_changes1 = hold_changes(:hold_changes1)
+    @hold_changes2 = hold_changes(:hold_changes2)
+    @hold2 = holds(:hold2)
+  end
+
+  test 'Update status of the hold' do
+    # Hold status is 'new' before update
+    assert_equal("new", @hold2.status)
+
+    @hold2.status = 'cancel'
+    @hold_changes1.update_hold
+    # Hold status is 'cancel' after update
+    assert_equal("cancel", @hold2.status)
+  end
+
+  test 'send hold change status email' do
+    # hold_change status should be ['error', 'pending', 'closed', 'cancelled'] than only sending status email.
+    @hold_changes1.status = 'closed'
+    resp = @hold_changes1.send_change_status_email
+    assert_equal("Order closed | Your teacher set order for MyString1", resp.subject)
+
+    # hold_change status is new not sending email
+    @hold_changes1.status = 'new'
+    resp = @hold_changes1.send_change_status_email
+    assert_nil(resp)
+  end
+
+
+  test 'send hold teacher_set deleted email' do
+    @hold_changes1.status = 'closed'
+    resp = @hold_changes1.send_teacher_set_deleted_email
+    assert_equal("Order closed | Teacher Set you requested has been deleted.", resp.subject)
```

Comment:
Could you just change this to "Order closed | The Teacher Set you requested  has been deleted", so adding 'The" and omitting the '.' at the end of the subject